### ISSUE TITLE
关于训练的疑问

### ISSUE THREAD

**Issue by E-Dreamer-LQ** [bubbliiiing/yolov4-pytorch#40]
您好，首先感谢您的分享。 我有几个疑问之处： 
1. 为什么先冻结25个epoch 的训练再解冻 再训练？ 
2. 为什么我训练前 25个epoch时, 大概 显存 9个g左右， 第26个epoch就直接显存炸了？ 我改小了 batchsize 也是不行。

**Comment by E-Dreamer-LQ** [comment]
@bubbliiiing

**Comment by bubbliiiing** [comment]
你用的什么显卡， batch是多少

**Comment by E-Dreamer-LQ** [comment]
用 的这个
![image](https://user-images.githubusercontent.com/38135798/87112964-ccd01900-c29f-11ea-9288-f5f4c928c075.png)

**Comment by E-Dreamer-LQ** [comment]
Quadro P5000    16G内存，　另外我用　训练出来model来算 map, 算出来是0， 用你的model能有0.8左右的map。 是哪里有问题么？ val loss 都收敛到很小了

**Comment by E-Dreamer-LQ** [comment]
batch现在最大能设定到 6 ，是直接可以注释冻结去训练了然后去算map 吧？

**Comment by bubbliiiing** [comment]
batch_size=6已经很大了，至于为什么是0的话，你是不是生成的2007_train.txt 图片后面没有物体信息

**Comment by sxpdwkj** [comment]
> batch现在最大能设定到 6 ，是直接可以注释冻结去训练了然后去算map 吧？

大佬，问题一模一样，一到26就爆了，按理说不应该啊，想问问你最后怎么弄的
